packages/labs/react/src/test/use-controller_test.ts

```diff
@@ -69,33 +69,63 @@ suite('useController', () => {
     );
   };
 
-  test('basic lifecycle', () => {
-    let testController!: TestController;
-
-    const TestComponent = ({x}: {x: number}) => {
-      testController = useTest('a');
-      return React.createElement('div', {className: 'foo'}, [
-        `x:${x}, a:${testController.a}`,
-      ]);
-    };
-
-    const render = (props: any) => {
-      ReactDOM.render(React.createElement(TestComponent, props), container);
-    };
-
-    render({x: 1});
-    assert.equal(ctorCallCount, 1);
-    assert.equal(container.innerHTML, `<div class="foo">x:1, a:a</div>`);
-    assert.deepEqual(testController.log, ['connected', 'update', 'updated']);
-    const firstTestController = testController;
+  const lifeCycleTest = ({strict}: {strict: boolean}) => {
```

Comment:
Why don't we just only test under strict mode?

Reply:
Seems important to test both since the behavior is different.